src/Runner.js

```diff
@@ -172,16 +161,63 @@ function run(transformFile, paths, options) {
         reject(e.message);
       });
     });
-  } else if (!fs.existsSync(transformFile)) {
-    process.stderr.write(
-      colors.white.bgRed('ERROR') + ' Transform file ' + transformFile + ' does not exist \n'
-    );
-    return;
-  } else {
-    return transform(transformFile);
+  } else if (fs.existsSync(transform)) {
+    let stats = fs.lstatSync(transform);
+
+    if (stats && stats.isFile(transform)) {
+      // handle file
+      return transform;
+    } else if (stats && stats.isDirectory(transform)) {
+      let directory = path.join(transform, 'transforms');
+      let file = path.join(transform, 'transformjs');
+
+      // handle directory
+      if (fs.existsSync(directory) && fs.lstatSync(directory).isDirectory()) {
+        return fs.readdirSync(directory);
```

Comment:
I feel like it might be worth to to be a more defensive here and filter out anything that is not a JS file (at least anything that is a directory).